bullet/src/FreeGroupFeatures.hh

```diff
@@ -0,0 +1,41 @@
+#ifndef IGNITION_PHYSICS_BULLET_SRC_FREEGROUPFEATURES_HH_
+#define IGNITION_PHYSICS_BULLET_SRC_FREEGROUPFEATURES_HH_
+
+#include <ignition/physics/FreeGroup.hh>
+
+#include "Base.hh"
+
+namespace ignition {
+namespace physics {
+namespace bullet {
+
+using FreeGroupFeatureList = FeatureList<
+  FindFreeGroupFeature,
+  SetFreeGroupWorldPose
+>;
+
+class FreeGroupFeatures
+    : public virtual Base,
+      public virtual Implements3d<FreeGroupFeatureList>
+{
+  // ----- FindFreeGroupFeature -----
+  Identity FindFreeGroupForModel(const Identity &_modelID) const override;
+
+  Identity FindFreeGroupForLink(const Identity &/* _linkID */) const override
```

Comment:
This and `GetFreeGroupCanonicalLink` seem to be a dummy implementation, did you check if we can add those before merging?

Reply:
Im not even sure if those 2 functions are used at all

Reply:
Update: Seem to be necessary for ignition, even if dummy, I'll look into implementing them

Reply:
9dafb2d Updates Canonical, but i need to finish that template (does not compile yet). Link has been added on c1e145c